docs/requirements-tutorials.txt

```diff
@@ -3,4 +3,4 @@ deep-phonemizer
 boto3
 pandas
 librosa
-numpy<=1.20
```

Comment:
Let's keep this as-is until `numba` supports `numpy >=1.21`.
Theoretically, this is only for building documentation, and it should not affect the requirements of a service using torchaudio.

Reply:
Sure. Let's wait the news from numba